cli/cmd/root.go

```diff
@@ -108,6 +108,20 @@ var rootCmd = &cobra.Command{
 				return err
 			}
 			setMountPointsForFamiliesInput(mountPoints, config)
+		} else {
+			// Check if volume already mounted
```

Comment:
This is too specific for the docker provider implementation. 
If a user wants to scan a path other than "/mnt/snapshot" he will fail

I think you need to pass the "/mnt/snapsho" as part of the families config as an input

Reply:
looking into these changes now :)

Reply:
@fishkerez please have a look at the last commit https://github.com/openclarity/vmclarity/pull/462/commits/61678c56aa60ea6a6c28d5307261f4011f60a035

Reply:
Looks good